MrtCore/winui_desktop_packaged_app/ClassLibrary/Class.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Text;
+using Microsoft.ApplicationModel.Resources;
+
+// To learn more about WinUI, the WinUI project structure,
+// and more about our project templates, see: http://aka.ms/winui-project-info.
+
+namespace ClassLibrary
+{
+    public class Class
+    {
+        public string GetClassLibrarySampleString()
+        {
+            var resourceLoader = new ResourceLoader();
+            var uri = new System.Uri("ms-resource://323858f2-34e1-48a5-8244-5c1bc0c1f889/ClassLibrary/ClassLibraryResources/ClassLibrarySampleString");

```

Comment:
Just curious, but what is this magic GUID (`323858f2-34e1-48a5-8244-5c1bc0c1f889`)?

Reply:
That's part of the full URI of the resource in the PRI index. It shouldn't be necessary in the URI / string for the lookup, but I couldn't get it to work without it for some reason (I kept getting element not found [for the URI based lookup] or NamedResource not found [for the string based lookup]). @axelandrejs or @huichen123 Do you know why that is?

Reply:
Actually, never mind. Addressed this in iteration 2.

Reply:
Ah, thanks for explaining! :)